google/cloud/bigquery/v2/minimal/internal/dataset.h

```diff
@@ -167,7 +221,7 @@ struct Dataset {
   ;
   std::chrono::hours max_time_travel = std::chrono::hours(0);
 
-  std::map<std::string, std::string> labels;
+  std::multimap<std::string, std::string> labels;
```

Comment:
If this isn't a change you would prefer to make, then I'm guessing it was because `DebugFormatter` doesn't support `std::map`.  We can address that if you'd like.  Let me know.